tests/conftest.py

```diff
@@ -307,85 +290,281 @@ def simple_inventory_node() -> None:
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_quantity_node() -> cript.Quantity:
-    """
-    minimal quantity node
-    """
-    my_quantity = cript.Quantity(key="mass", value=1.23, unit="gram")
+    quantity = cript.Quantity("mass", 11.2, "kg", 0.2, "std")
+    return quantity
 
-    return my_quantity
+
+@pytest.fixture(scope="function")
+def simple_quantity_dict() -> dict:
+    ret_dict = {"node": ["Quantity"], "key": "mass", "value": 11.2, "unit": "kg", "uncertainty": 0.2, "uncertainty_type": "std"}
+    return ret_dict
+
+
+@pytest.fixture(scope="function")
+def simple_software_node() -> cript.Software:
+    software = cript.Software("SOMA", "0.7.0", "https://gitlab.com/InnocentBug/SOMA")
+    return software
+
+
+@pytest.fixture(scope="function")
+def simple_software_dict() -> dict:
+    ret_dict = {"node": ["Software"], "name": "SOMA", "version": "0.7.0", "source": "https://gitlab.com/InnocentBug/SOMA"}
+    return ret_dict
+
+
+@pytest.fixture(scope="function")
+def simple_property_node(simple_material_node, simple_condition_node, simple_citation_node, simple_data_node, simple_process_node, simple_computation_node):
+    p = cript.Property(
+        "modulus_shear",
+        "value",
+        5.0,
+        "GPa",
+        0.1,
+        "std",
+        structure="structure",
+        method="method",
+        sample_preparation=[simple_process_node],
+        conditions=[simple_condition_node],
+        computations=[simple_computation_node],
+        citations=[simple_citation_node],
+        notes="notes",
+    )
+    return p
+
+
+@pytest.fixture(scope="function")
+def simple_property_dict(simple_material_node, simple_condition_dict, simple_citation_dict, simple_data_node, simple_process_node, simple_computation_node) -> dict:
+    ret_dict = {
+        "node": ["Property"],
+        "key": "modulus_shear",
+        "type": "value",
+        "value": 5.0,
+        "unit": "GPa",
+        "uncertainty": 0.1,
+        "uncertainty_type": "std",
+        "structure": "structure",
+        "sample_preparation": [json.loads(simple_process_node.json)],
+        "method": "method",
+        "conditions": [simple_condition_dict],
+        "citations": [simple_citation_dict],
+        "computations": [json.loads(simple_computation_node.json)],
+        "notes": "notes",
+    }
+    return strip_uid_from_dict(ret_dict)
+
+
+@pytest.fixture(scope="function")
+def simple_condition_node(simple_material_node, simple_data_node) -> cript.Condition:
+    c = cript.Condition(
+        "temp",
+        "value",
+        22,
+        "C",
+        "room temperature of lab",
+        uncertainty=5,
+        uncertainty_type="var",
+        set_id=0,
+        measurement_id=2,
+        material=[simple_material_node],
+        data=simple_data_node,
+    )
+    return c
+
+
+@pytest.fixture(scope="function")
+def simple_condition_dict(simple_material_node, simple_data_node) -> dict:
+    ret_dict = {
```

Comment:
I think `condition_dict` would have been a more self documenting name here

Reply:
This is testing, so I don't care as much about readability and documentation.
Since this is inside a fixture with a telling name, I am leaning towards readable enough here.